launcher/src/main/scala/bloop/launcher/package.scala

```diff
@@ -10,10 +10,19 @@ package object launcher {
   def printQuoted(msg: String, out: PrintStream): Unit = {
     println(
       msg
-        .split(System.lineSeparator())
+        .split("\n|\r\n|\r", -1)
```

Comment:
```suggestion
        .split("\r\n|\n", -1)
```
?

Reply:
I got rid of the -1 as well, since that possibly generates a trailing empty line, whereas the original "string".split(lineSeparator) did not.